src/flag-evaluation-details-builder.ts

```diff
@@ -7,6 +7,7 @@ export const flagEvaluationCodes = [
   'TYPE_MISMATCH',
   'ASSIGNMENT_ERROR',
   'DEFAULT_ALLOCATION_NULL',
+  'NO_BANDIT_ACTIONS_SUPPLIED',
```

Comment:
The `NO_BANDIT_ACTIONS_SUPPLIED` was meant to mean "No actions were supplied for the bandit flag that you're trying to use", but I could see how this might be misinterpreted as "Bandit actions are always required, and you didn't supply them". Maybe I'll rename to `NO_ACTIONS_SUPPLIED_FOR_BANDIT` to avoid confusion.